code/modules/mob/living/living.dm

```diff
@@ -799,13 +807,6 @@ Thanks.
 	on_resist = new(owner = src)
 	on_life = new(owner = src)
```

Comment:
I don't think having these here breaks anything, but while you're combining the two `Destroy()` procs you might as well combine the `New()` as well

Reply:
Sure